source/read/specify-documents-to-return.txt

```diff
@@ -0,0 +1,202 @@
+.. _php-specify-documents-to-return:
+
+===========================
+Specify Documents to Return
+===========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: read, paginate, pagination, order, code example
+
+Overview
+--------
+
+In this guide, you can learn how to specify which documents to return from a read
+operation by passing the following options to the ``MongoDB\Collection::find()`` or
+``MongoDB\Collection::findOne()`` method:
+
+- :ref:`limit <php-return-documents-limit>`: Specifies the maximum number of documents
+  to return from a query.
+- :ref:`sort <php-return-documents-sort>`: Specifies the sort order for the returned documents.
+- :ref:`skip <php-return-documents-skip>`: Specifies the number of documents to skip before
+  returning query results.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``restaurants`` collection in the ``sample_restaurants``
+database from the :atlas:`Atlas sample datasets </sample-data>`. To access this collection
+from your PHP application, instantiate a ``MongoDB\Client`` that connects to an Atlas cluster
+and assign the following values to your ``db`` and ``collection`` variables:
+
+.. literalinclude:: /includes/read/limit-skip-sort.php
+    :language: php
+    :dedent:
+    :start-after: start-db-coll
+    :end-before: end-db-coll
+
+To learn how to create a free MongoDB Atlas cluster and load the sample datasets, see the
+:atlas:`Get Started with Atlas </getting-started>` guide.
+
+.. _php-return-documents-limit:
+
+Limit
+-----
+
+To specify the maximum number of documents returned from a read operation, create
+an array that sets the ``limit`` option and pass the array as a parameter to the
+``MongoDB\Collection::find()`` method.
+
+The following example finds all restaurants that have a ``cuisine`` field value
+of ``'Italian'`` and limits the results to ``5`` documents:
```

Comment:
Q: I've seen other repos format this as ``cuisine`` field value of ``"Italian"`` (instead of ``'Italian'``). Is this a PHP-specific thing of how strings are enclosed with single quotes as opposed to double quotes?

Reply:
Yeah, I used single quotes to match the code example text (since PHP uses single quotes)